working_with_data/working_with_data.bal

```diff
@@ -9,17 +9,20 @@ type Country record {
     int deaths;
 };
 
-// Prints the top 10 countries having the highest case-fatality ratio.
+// Prints the top 10 countries having the highest case-fatality ratio grouped by continent.
```

Comment:
Let's not change this example, at least right now. It is linked in Ballerina in action page as well.